crates/vm/levm/src/precompiles.rs

```diff
@@ -186,8 +191,12 @@ pub fn ecrecover(
 
     // signature is made up of the parameters r and s
     let sig = calldata.get(64..128).ok_or(InternalError::SlicingError)?;
-    let signature =
-        Signature::parse_standard_slice(sig).map_err(|_| PrecompileError::ParsingInputError)?;
+    let signature = match Signature::parse_standard_slice(sig) {
+        Ok(sig) => sig,
+        Err(_) => {
+            return Ok(Bytes::new());
+        }
+    };
```

Comment:
Ditto
```suggestion
        let Ok(signature) = Signature::parse_standard_slice(sig) else {
        return Ok(Bytes::new());
    };

```

Reply:
```rust
let signature = Signature::parse_standard_slice(sig).unwrap_or_else(|_| return Ok(Bytes::new()));
```
is also an option if you like it

Reply:
Done [aef49b3](https://github.com/lambdaclass/ethrex/pull/1570/commits/aef49b32bedac0594c3ae281fb4eb8a372fe59fa)!

Reply:
Oops, in my mind it worked fine haha. Sorry!